src/WebJobs.Script/Config/ScriptHostConfiguration.cs

```diff
@@ -50,6 +51,14 @@ public ScriptHostConfiguration()
         /// The default is false. When set to true, logs will be written to the directory
         /// specified by <see cref="RootLogPath"/>.
         /// </summary>
-        public bool FileLoggingEnabled { get; set; }    
+        public bool FileLoggingEnabled { get; set; }
+
+        /// <summary>
+        /// Gets the list of functions that should be run. This list can be used to filter
+        /// the set of functions that will be enabled - it can be a subset of the actual
+        /// function directories. When left null (the default) all discovered functions will
+        /// be run.
+        /// </summary>
+        public Collection<string> Functions { get; set; }
```

Comment:
I guess we can keep that as an experimental thing to help our testing. We can decide later whether we want to call it an official feature. Not so likely to be useful outside of local debugging, since functions are in theory independent (from user standpoint).